WebRTC Auto Updater/Detect - WebRTC - Detection Script - 0.2.3.ps1

```diff
@@ -0,0 +1,138 @@
+﻿<#
```

Comment:
I remember you already submitted a PR two weeks ago to add these two scripts correct me if I'm wrong, could you help me understand what's the difference those two scripts with these files https://github.com/microsoft/Windows365-PSScripts/tree/main/WebRTC%20Auto%20Updater?

Reply:
You could click code tab like screenshot shows, then it will show all scripts you added in your last PR.
![image](https://user-images.githubusercontent.com/72924235/161284016-14c1fb3f-9561-4f10-8f14-f7f5b1fdf0fc.png)

Reply:
I just cloned the upstream repo (master) as the starting point. Should I have not done that?